mod_tariffs.asciidoc

```diff
@@ -674,73 +800,79 @@ Example of cost:
-|step_size |int |1 |Minimum amount to be billed. This unit will be billed in this step_size blocks. For example: if type is time and step_size is 300, then time will be billed in blocks of 5 minutes, so if 6 minutes is used, 10 minutes (2 blocks of step_size) will be billed.
+|type |<<mod_tariffs_tariffdimensiontype_enum,TariffDimensionType>> |1 |Type of tariff dimension.
+|price |<<types.asciidoc#types_number_type,number>> |1 |Price per unit (excl. VAT) for this tariff dimension.
+|vat |<<types.asciidoc#types_number_type,number>> |? |Applicable VAT percentage for this tariff dimension. If omitted, no VAT is applicable. Not providing a VAT is different from 0% VAT, which would be a value of 0.0 here.
+|step_size |int |1 |Minimum amount to be billed. This unit will be billed in this `step_size` blocks. For example: if `type` is `TIME` and `step_size` has a value of `300`, then time will be billed in blocks of 5 minutes. If 6 minutes were used, 10 minutes (2 blocks of `step_size`) will be billed.
 |===
 
-NOTE: `step_size`: depends on the `type`, every `type` (except `FLAT`) defines a step_size multiplier, this is the size of every 'step' for that type in the gaven unit. +
+NOTE: `step_size`: depends on the `type` and every `type` (except `FLAT`) defines a `step_size` multiplier, which is the size of every _step_ for that `type` in the givgen unit. +
  +
-For example: `PARKING_TIME` has 'step_size multiplier: 1 second' That means that the `step_size` of a `PriceComponent` is multiplied by 1 second.
-Thus a `step_size = 300` means 300 seconds = 5 minutes, this means that when somebody has parked for 8 minutes he will be billed for 10, parked for 11 minutes then billed for 15. (steps of 5 minutes) +
+For example: `PARKING_TIME` has the `step_size` multiplier: _1 second_, which means that the `step_size` of a `PriceComponent` is multiplied by _1 second_.
+Thus a `step_size = 300` means `300 seconds` (`5 minutes`). This means that when someone parked for 8 minutes they will be billed for 10 minutes.
+The parking time will be simply rounded up to the next larger chunk of `step_size` (i.e. blocks of `300 seconds` in this example). +
  +
-Example with `ENERGY`: `Price`= 0.25, `step_size` = 1. +
-Then energy is billed per 1 Wh. If somebody charges his EV with 115.2 Wh, then he is billed for 116 Wh, price: 0.029. +
-When `step_size` = 25, then the same amount will be billed for 125 Wh, price: 0.031. +
-When `step_size` = 500, then the same amount will be billed for 500 Wh, price: 0.125.
+Another example: `ENERGY` has the `step_size` multiplied: _1 Wh_, which means that the `step_size` of a `PriceComponent` is multiplied by _1 Wh_.
+Thus a `step_size = 1` with a `price = 0.25` will result in a cost calculation that uses the charged Wh as precision. +
+If someone charges their EV with 115.2 Wh, then they are billed for 116 Wh, resulting in total cost of 0.029 euro. +
```

Comment:
I did add a hint at the beginning of the examples section in 2c75043 as it affects all of the examples. I made the hint future proof so that we can also add other-currency-examples if necessary without producing confusion.